bitbucket-pipelines.yml

```diff
@@ -61,15 +61,15 @@ pipelines:
       - step: 
           name: Dry Run TFY Changes
           script:
-            - apt-get update && apt-get install -y python3-pip && apt-get install -y git
+            - apt-get update && apt-get install -y git
```

Comment:
are we sure pip is pre-installed?

Reply:
yes, tested this. Using python 3.8 as base image